robots.txt

```diff
@@ -0,0 +1,4 @@
+User-agent: *
+Disallow: /admin/
+Disallow: /private/
```

Comment:
I had a question on this, from another Pull Request here: https://github.com/nathandh/mywebclass-simulation/pull/39#discussion_r1142867638

Also, you might have merge conflicts at some point when you create multiple branches and make similar/same changes. In this case, it might have been recommended to have changes staged and ready once the other CR that adds this same ``robots.txt`` file was approved and merged. 

Or, since that is not always possible. Maybe isolate your changes in smaller chunks so that you dont need the duplicated code. *In this case, the duplicated robots.txt file that is in another CR.